apps/frontend/src/pages/Homepage.tsx

```diff
@@ -128,7 +128,10 @@ const Homepage = () => {
       if (initialRender.current) {
         initialRender.current = false;
         _axios_instance
-          .get("/recommendations", { params: { page: 1, limit: 24 } })
+          // remove hardcoded authorization once it can correctly be handled/after Sprint2 demo.
+          .get("/recommendations", {
+            params: { authorization: 5, page: 1, limit: 24 },
```

Comment:
a whole bunch of backend stuff uses auth 1, I'm wondering if we should set it to that.